README.md

```diff
@@ -18,11 +18,43 @@ jobs:
         anaconda_nightly_upload_token: ${{secrets.UPLOAD_TOKEN}}
 ```
 
-Note that we recommend pinning the action against a specific SHA
+> [!IMPORTANT]
+> Note that we recommend pinning the action against a specific SHA
 (rather than a tag), to guard against the unlikely event of upstream
 being compromised.
 
-## Updating the action
+# Removing old nightly builds
+
+This repository also ships with an action to ease removals of older nightly wheels from a channel.
+
+To use this functionality, add the following snippet to your workflow:
+
+```yml
+jobs:
+  steps:
+    ...
+    - name: Remove old wheels
+      uses: scientific-python/upload-nightly-action/remove-wheels@cantknowhashyet # 0.6.0
+      with:
+        n_latest_uploads: ${{ env.N_LATEST_UPLOADS }}
+        anaconda_nightly_upload_organization: "your-organization"
```

Comment:
This is called organization, not channel; are these interchangeable terms? If so, use channel; if not, explain the difference.

Reply:
I'm not quite sure, since they look like they have been used interchangeably. For example, the README outside of this PR:

https://github.com/scientific-python/upload-nightly-action/blob/920fb594ddba437d939052cabd2a59ceb67df265/README.md?plain=1#L48-L67

mentions how one may upload to a different channel, but it uses "organization" as an input.